### DISCUSSION TITLE
Share you code here.

### DISCUSSION THREAD

**Discussion by War3Evo** [War3Evo/War3Source-EVO#38] [Ideas]
If you have choose you'd like to share.  Drop the files here.

**Comment by BASON73** [comment]
[ultimates-spells.zip](https://github.com/War3Evo/War3Source-EVO/files/10771397/ultimates-spells.zip)

There are quite a lot of different skills as well as ultimates if it helped you, I will be glad

**Reply by War3Evo** [reply]
Those are very large files.  I am sure they would become useful.

Do you also have the sounds for those in a zip file?

**Comment by BASON73** [comment]
[sound.zip](https://github.com/War3Evo/War3Source-EVO/files/10771561/sound.zip)

**Reply by War3Evo** [reply]
Ok.  Thank you.

It'll be around next weekend that I'll begin incorporating those into races and or building a configuration system so that races can be created via a configuration file system.

It would be however, better to just create 1 compiled file per race, instead of a configuration file.  I will study it and figure out the best method.

Reason I'm thinking of using a compiled file per race instead of configuration file is because I can isolate the races not being used from memory and saving processing power for lower end servers.
